zboxcore/sdk/commitworker.go

```diff
@@ -143,6 +143,11 @@ func (commitreq *CommitRequest) processCommit() {
 	}
 
 	if lR.LatestWM != nil {
+		if lR.LatestWM.ClientID != client.GetClientID() {
```

Comment:
This check seems useful for debug purpose only.
Signature verification will also prove if client id is same or not